lib/parse/push.rb

```diff
@@ -17,32 +16,35 @@ class Push
     def initialize(data, channel = '', client = nil)
       @data = data
 
+      # NOTE: if no channel is specified, by setting "where" to an empty clause
+      #   a push is sent to all clients.
       if !channel || channel.empty?
-        # If no channel is specified, by setting "where" to an empty clause, a push is sent to all clients.
         @where = {}
       else
-        @channel = channel
+        @channels = [channel]
```

Comment:
I don't think we should change the public interface of the initializer nor it should be a catch all (which actually adds ambiguity and I'm trying to remove it). It works like this:

- if you need to target one channel you can pass the string to `initialize` and be done with it
- if you have a list of channel just use the `channels` attribute and provide a list
- the channel you provide ends up in the `channels` attribute anyway so you have only *one place* to look (instead of `channel` and `channels` right now) to know which channels the push is targeting.

Channel defaults to an empty string because it accepts a string. It seemed the obvious choice between the two.